pkg/kubelet/metrics/metrics.go

```diff
@@ -917,6 +921,28 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// DRAPrepareResourcesDuration tracks the duration of the DRA PrepareResources request.
+	DRAPrepareResourcesDuration = metrics.NewHistogram(
+		&metrics.HistogramOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           DRAPrepareResourcesDurationKey,
+			Help:           "Duration in seconds of the DRA PrepareResources request.",
+			Buckets:        metrics.DefBuckets,
+			StabilityLevel: metrics.ALPHA,
+		},
+	)
+
+	// DRAUnprepareResourcesDuration tracks the duration of the DRA UnprepareResources request.
+	DRAUnprepareResourcesDuration = metrics.NewHistogram(
```

Comment:
same here

Reply:
Thank you for the review! Should be done now.